### DISCUSSION TITLE
On the Replies tab of account profiles, filter out all non-reply posts

### DISCUSSION THREAD

**Discussion by venteto** [bluesky-social/atproto#1999] [Bluesky Client App]
This was something I never understod with Twitter, posts being mixed in with replies. I can already see non-reply posts under the Posts tab. Clicking something else usually means I want to do something else that does not duplicate the output of another clickable item.

This is another manifestation of redundancy in the client app, carried over from Twitter.

**Comment by surfdude29** [comment]
I think this would be rather unintuitive to most users. Like on Twitter, I expect the Replies tab to actually be a Posts & Replies tab and for it to show me **all** posts that have been made by the user, not just replies to other users (the Twitter URL is 'with_replies' which is actually clearer than just calling it Replies).

In other words, the Posts tab should be a subset of the Replies tab imo, as it currently is on both Bluesky and Twitter.
